src/schemas/meta_transaction_quote_request_schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "id": "/MetaTransactionQuoteRequestSchema",
+    "properties": {
+        "sellToken": {
+            "type": "string"
+        },
+        "buyToken": {
+            "type": "string"
+        },
+        "takerAddress": {
+            "$ref": "/addressSchema"
+        },
+        "slippage": {
```

Comment:
We've been using `slippagePercentage` as a query param.

Reply:
I copied `SwapQuoteRequestSchema`. Is that incorrect as well?

Reply:
Yeah looks like it.